tchannel/thrift/generated.py

```diff
@@ -0,0 +1,95 @@
+from __future__ import (
+    absolute_import, division, print_function, unicode_literals
+)
+
+import inspect
+import types
+
+from .util import get_service_methods
+
+
+def from_thrift_module(service, thrift_module):
```

Comment:
this might be better if api was like:

```
from tchannel import thrift

service = thrift.from_module(thrift_module)
service = thrift.from_idl('some_idl.thrift')
```